components/create_post/create_post.test.jsx

```diff
@@ -888,16 +874,30 @@ describe('components/create_post', () => {
     });
 
     it('Should call Shortcut modal on FORWARD_SLASH+cntrl/meta', () => {
-        const wrapper = shallowWithIntl(createPost());
+        const openModal = jest.fn();
```

Comment:
You actually don't need to redefine this here. You can just check if `actionsProp.openModal` has been called below exactly the same as you're doing now. Even though other tests use the same mock, it's automatically reset between test cases

Reply:
yes i removed redundant mocking again for `openModal`